File: swift/입문/머쓱이보다_키_큰_사람.swift
import Foundation

func solution(_ array:[Int], _ height:Int) -> Int {
    var array_height = array
    array_height.append(height)
    array_height.sort()
    if let height_index = array_height.lastIndex(of: height) {
        return array_height.count - height_index - 1
    }
    return 0
}

File: python/입문/머쓱이보다_키_큰_사람.py
def solution(array, height):
    answer = list(filter(lambda x: x > height, array))
    return len(answer)

# len vs count 학습하기
# 제네레이터 컴프리헨션 학습하기
# def solution(array, height):
#     return sum(1 for a in array if a > height)